app/admin/practices.rb

```diff
@@ -20,14 +20,15 @@
   end
 
   index do
-    Practice.create
+    #Practice.create
```

Comment:
honestly not sure why this was here before but if we don't need it we can just remove

Reply:
Yeah  for some reason this was creating a blank practice when just viewing the practice... not sure why it was there either.. I'll remove it..

Reply:
Thank you!!